psydac/fem/partitioning.py

```diff
@@ -231,6 +231,26 @@ def construct_interface_spaces(domain_decomposition, g_spaces, carts, interiors,
         g_spaces[interiors[i]].create_interface_space(axis_minus, ext_minus, cart=cart_minus)
         g_spaces[interiors[j]].create_interface_space(axis_plus , ext_plus , cart=cart_plus)
 
+        max_ncells = tuple(max(ni,nj) for ni,nj in zip(g_spaces[interiors[i]].ncells,g_spaces[interiors[j]].ncells))
+        cart_minus = g_spaces[interiors[i]]._refined_space[tuple(max_ncells)].vector_space.cart
+        cart_plus  = g_spaces[interiors[j]]._refined_space[tuple(max_ncells)].vector_space.cart
```

Comment:
We should not read the private property `_refined_space`.

Reply:
done in [f74fd12](https://github.com/pyccel/psydac/pull/241/commits/f74fd122c1b075e8f616d556a9b44aa8408bb4a1)